apps/web/lib/swr/use-workspace-store.ts

```diff
@@ -6,13 +6,13 @@ import { mutate } from "swr";
 import { updateWorkspaceStore } from "../actions/update-workspace-store";
 import useWorkspace from "./use-workspace";
 
-export const STORE_KEYS = {
-  conversionsOnboarding: "conversionsOnboarding",
-  dotLinkOfferDismissed: "dotLinkOfferDismissed",
-};
+type STORE_KEYS =
+  | "conversionsOnboarding"
+  | "dotLinkOfferDismissed"
+  | "programOnboarding";
```

Comment:
Can we infer this from `updateWorkspaceStoreSchema` (or a separate shared schema) so we don't need to update these keys in multiple places?